version/protocol_versions.go

```diff
@@ -28,5 +31,9 @@ func ConfigureProtocolVersions(network string) (*ProtocolVersionTable, error) {
 		Add(DEVNET4, Protocol0, types.NewBlockHeight(0)).
 		Add(LOCALNET, Protocol0, types.NewBlockHeight(0)).
 		Add(TEST, Protocol0, types.NewBlockHeight(0)).
+		Add(USER, Protocol1, types.NewBlockHeight(40000)).
+		Add(DEVNET4, Protocol1, types.NewBlockHeight(300)).
+		Add(LOCALNET, Protocol1, types.NewBlockHeight(300)).
+		Add(TEST, Protocol1, types.NewBlockHeight(300)).
```

Comment:
For almost all tests I would expect to use version 1 from the beginning. Only the tests specifically exercising both behaviours should have a non-zero upgrade height.

Don't call into this method from those tests if this isn't building the right table - just build a custom table with only the TEST network. Reserve this common table for "shared" networks.

So I expect this table to be:
USER: Protocol0, 0
USER: Protocol1, 40000
LOCALNET: Protocol1, 0
TEST: Protocol1, 0

@acruikshank